api/index.js

```diff
@@ -0,0 +1,42 @@
+const config = require('../config');
+const message = config.message;
+const helper = require('../helper');
+const responseHelper = helper.response;
+const service = require('../service');
+const _ = require('lodash');
+
+module.exports = {
+    linkAccount: (request, response) => {
+        const token = request.body;
+
+        if (_.isEmpty(token)) {
```

Comment:
instead of using if/else, do 

if(_.isEmpty...){
   return responseHandler.failure(...)
}

Makes it a bit cleaner and we don't have to use all these brackets and indenting :)